conf/defaults.ini

```diff
@@ -397,6 +397,7 @@ global_session = -1
 [alerting]
 # Makes it possible to turn off alert rule execution.
 execute_alerts = true
+show_alerting_ui = true
```

Comment:
Actually @bergquist is right, in the current Grafana setup where alerting_engine can be enabled only one instance, we need two flags in the config. One for enabling/disabling alerting_engine and another for enabling/disabling alerts_configuration_in ui.